### PR TITLE
 Limit audit logs retrieve days by feature config

### PR DESCRIPTION
**Pull request by carmenlau**
require #1372

### CODE REVIEW

File: pkg/lib/config/feature_audit_log.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package config
+
+var _ = FeatureConfigSchema.Add("AuditLogFeatureConfig", `
+{
+	"type": "object",
+	"additionalProperties": false,
+	"properties": {
+		"retrieve_days": { "type": "integer" }
```
Comment (louischan-oursky) [inline pkg/lib/config/feature_audit_log.go:8]: `retrieval_days` seems more grammatically correct.

**Review by louischan-oursky — COMMENTED** [review]
We should also appear the limit to the date range control.

**Comment by carmenlau** [comment]
Updated the config name and added date boundaries to audit log date picker

**Comment by carmenlau** [comment]
Updated to set the from query for full date range

**Pull request merged**